packages/reporter/src/header/header.tsx

```diff
@@ -20,11 +21,20 @@ export interface ReporterHeaderProps {
 
 const Header = observer(({ appState, events = defaultEvents, statsStore }: ReporterHeaderProps) => (
   <header>
-    <Tooltip placement='bottom' title={<p>View All Tests <span className='kbd'>F</span></p>} wrapperClassName='focus-tests' className='cy-tooltip'>
-      <button onClick={() => events.emit('focus:tests')}>
-        <MenuExpandRightIcon />
+    <Tooltip placement='bottom' title={<p>View All Specs <span className='kbd'>F</span></p>} wrapperClassName='toggle-specs-wrapper' className='cy-tooltip'>
```

Comment:
We discussed this already, but mentioning it here too: Please change the tooltip to toggle between “Expand Specs List” and “Collapse Specs List”. Sorry I missed this when I was doing the initial work for this ticket!